packages/flutter/lib/src/widgets/binding.dart

```diff
@@ -1152,21 +1156,28 @@ mixin WidgetsBinding on BindingBase, ServicesBinding, SchedulerBinding, GestureB
   }
 }
 
-/// Inflate the given widget and attach it to the screen.
+/// Inflate the given widget and attach it to the view.
+///
+// TODO(goderbauer): Update the paragraph bellow to include the Window widget once that exists.
+/// The [runApp] method renders the provided `app` widget into the
+/// [PlatformDispatcher.implicitView] by wrapping it in a [View] widget, which
+/// will bootstrap the render tree for the app. Apps that want to control which
+/// [FlutterView] they render into may use [runWidget] instead.
```

Comment:
I think `can` or `should` would be preferrable here (`may` indicates permission).

```suggestion
/// [FlutterView] they render into can use [runWidget] instead.
```